src/js/controllers/imageCtrl.js

```diff
@@ -0,0 +1,27 @@
+(function (angular) {
+    angular
+        .module('flickrPOC')
+        .controller('imagesCtrl', imagesCtrl);
+
+    imagesCtrl.$inject = ['images', '$log'];
+
+    function imagesCtrl(images, $log) {
+        var vm = this;// jshint ignore:line
+
+        function activate(){
+
+            images.loadPics()
+                .then(function (res) {
+                    vm.images = res;
+                    return vm.images;
+                })
+                .catch(function(err){
+                    $log.error('Error loading images ', err.status, err.data);
+                });
+        }
+
+        activate();
+
+    }
+
+})(angular);
```

Comment:
One additional note, look at the difference in number of lines of code between the Controller (27) and the Tests (142). It may seem like that tests are overkill and would take up too much time to write and maintain, however, what I see is a solid controller that I can feel comfortable using and modifying.